### DISCUSSION TITLE
Can't install Homebrew? (sorry total noob question)

### DISCUSSION THREAD

**Discussion by albrbr** [Homebrew/discussions#128] [Everyday usage] [answered]
Hi, 

I decided to try to learn how to code with python, and am trying to follow introtopython.org's curriculum. I've managed to get stuck on the set-up even before the curriculum starts. I went to homebrew's website, which wants me to install homebrew by pasting: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)" in terminal. When I do so, a new line appears that says "Password" with an image of a key next to it. I guessed this meant I had to type my administrator password in to download the software. However, when I started typing my password on my keyboard, nothing happens / shows up on the screen. If I press enter it registers as a failed attempt, but none of my letter presses do anything. What should I do in order to input my password / finish downloading homebrew?

**Comment by Rylan12** [comment, marked as answer]
Hi there!

This is normal: the terminal won't display anything to protect your password, but it will still register the keystrokes. If you type your password and then hit enter, even though nothing has shown up, the script should continue.

**Reply by albrbr** [reply]
haha I feel like a fool thank you very much for the fast response

**Reply by Rylan12** [reply]
No worries! We've all been there!

**Reply by aidahasanova11** [reply]
Couldn't find in many YT videos. Thank you!
